### PR TITLE
Feature/aut 963/e2e testing item add interactions to canvas

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/AUT-963

Added commands to cypress `dragAndDrop` and `fileUpload`

### CODE REVIEW

**Review by Silvia-loza — APPROVED** [review]
Complex addition to e2e initiative, it looks good and all tests pass consistently for all interactions. Great job!  
- [ ] New code is covered by tests (if applicable)
- [X] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

**Review by AnastasiaArcadia — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [X] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X] New code is not subject to concurrency issues (if applicable)
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

File: views/cypress/support/file-upload.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ */
+
+import '@oat-sa/e2e-runner/support/fileupload';
+
+/**
+ * Upload file to input
+ * @param {String} importSelector -  selector for input type="file"
+ * @param {String} importFilePath  - parh to file
+ */
+Cypress.Commands.add('fileUpload', (importSelector, importFilePath) => {
```
Comment (AnastasiaArcadia) [inline views/cypress/support/file-upload.js:26]: Can we use this command in import / export items tests?

Comment (bziondik) [reply views/cypress/support/file-upload.js:26]: yes, why not?

**Review by clagosarias — APPROVED** [review]
Awesome job!

- [ ] New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [x]  The feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request's target is not master
- [x]  Pull request title and description are meaningful
- [x]  Commits are following conventional commits
- [x]  Kitchen links are not exposed in the description

**Pull request merged**